--- scripts/makepkg.sh.in	2015-02-14 02:30:05.000000000 +0100
+++ makepkg.sh.in	2015-08-22 12:29:00.000000000 +0200
@@ -1407,7 +1407,7 @@
 	local file ext decompress found pubkey success status fingerprint trusted
 	local warning=0
 	local errors=0
-	local statusfile=$(mktemp)
+	local statusfile=$(mktemp /tmp/makepkg.XXXXXX)
 	local all_sources
 
 	case $1 in
@@ -3431,11 +3431,6 @@
 catastrophic damage to your system.")" "makepkg"
 		exit 1 # $E_USER_ABORT
 	fi
-else
-	if [[ -z $FAKEROOTKEY ]]; then
-		error "$(gettext "Do not use the %s option. This option is only for use by %s.")" "'-F'" "makepkg"
-		exit 1 # TODO: error code
-	fi
 fi
 
 unset pkgname pkgbase pkgver pkgrel epoch pkgdesc url license groups provides
